src/weakrates/ratelibrary.F90

```diff
@@ -18,7 +18,7 @@ module class_ratelibrary
   end interface new_RateLibrary
 
   interface return_weakrate
-     module procedure return_weakrate_dynamic_search, return_weakrate_from_table, return_weakrate_from_approx
+     module procedure return_weakrate_dynamic_search, return_weakrate_from_table, return_weakrate_from_approx, return_weakrate_from_approx2
```

Comment:
After we make the change to how the approximation mode is specified we can get rid of the old approximation.